styleguide/README.md

```diff
@@ -0,0 +1,55 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# Style Guide
+
+Below are a set of conventions contributors should use to write code and documentation. Using a common set of standards make it easier for others to learn and contribute their ideas over time.
+
+# Terminology
+
+* The [Vendia Share web application](https://share.vendia.net) should be referred to as `Vendia Share web application` or `Vendia Share web app`.
+
+* The [Vendia Share command line application (CLI)](https://www.vendia.net/docs/share/cli) should be referred to as the `Vendia Share CLI` or `Share CLI`.
+
+* Parties that comprise a Uni should be referred to as `participants`.
+
+* The term `Universal Application` and the abbreviated `Uni` should be capitalized as a proper noun.
+
+* The word `node` should be lower-case.
+
+* The word `participant` should be lower-case.
+
+* Use [camel-case](https://wiki.c2.com/?CamelCase) when referring to a _specific_ `participant` (e.g. `ShippingCompany`).
+
+* Use [camel-case](https://wiki.c2.com/?CamelCase) when referring to the `node` of a _specific_ `participant`. The name of the `node` should be **bold** (e.g. **ShippingCompanyNode**).
+
+# Writing Style
+
+* Use the following convention when adding a **NOTE:** to a document.
+
+```
+**NOTE:** This is something that should be brought to a user's attention
+```
+
+* Use the following convention when adding a **WARNING:** to a document
+
+```
+**WARNING:** This is a warning of a potentially dangerous action.
+```
+
+* Code snippets should be included in matching \`\`\` characters.
+
+```
+echo "This is sample code."
+```
+
+* Use the pronoun "you" and the the future tense "will" when providing instructions to readers. Contractions are ok - we're not monsters. ;-)
+
+# Guidelines for Content Length
+
+## Blogs
+
+The ideal blog post length should be 2100 - 2400 words [according to HubSpot](https://blog.hubspot.com/marketing/how-long-should-your-blog-posts-be-faq).
```

Comment:
I think Caroll said recently long-form blogs were gaining favor.  I'm not an expert but curious if she and TZ have an opinion on content length guidance.